File: trunk/quests/qeynos/Earron_Huntlan.pl
#Earron_Huntlan.pl
#Kwint's Kwest
#Details on what he says when you hand him back the scrap, if you choose to, are non-existant, as are the supposed different faction hits than giving it to Kwint. -Kilelen

sub EVENT_SAY {
	if($text=~/Kwint sent me/i)   {
		quest::say("Kwint? Gave you the line about being my brother, huh? That boy sends me more customers, I tells ya! I give him one free drink for every person he gets to come in. Hey, could you do me a favor and take this voucher to him when you are done drinking? Thanks.");
	}
}

sub EVENT_ITEM {
	if (plugin::check_handin(\%itemcount, 18821 => 1)) {#Scrap of Parchment
		quest::say("");#text made up
		quest::summonitem(13039);#Ale
		#Supposed to be different faction hits here, but it's not mentioned what, so not giving any.
	}
	
	else {
		plugin::return_items(\%itemcount);
	}
}

#End of File zone:qeynos ID: 1106 -- Earron_Huntlan.pl

File: qeynos/Earron_Huntlan.lua
-- items: 18821
function event_say(e)
	if(e.message:findi("hail")) then
		e.self:Say("Greetings, thirsty adventurer! Please help yourself to the free salty snacks. We are here to serve you at the Lion's Main Tavern. Allow yourself to indulge from our wide selection oof fine spirits.");
	elseif(e.message:findi("kwint sent")) then
		e.self:Say("Kwint? Gave you the line about being my brother, huh? That boy sends me more customers, I tells ya! I give him one free drink for every person he gets to come in. Hey, could you do me a favor and take this voucher to him when you are done drinking? Thanks.");
		e.other:SummonItem(18821); -- Item: Scrap of Parchment
	end
end

function event_trade(e)
	local item_lib =require("items");
	if(item_lib.check_turn_in(e.trade, {item1 = 18821})) then
		e.self:Say("What the..? This voucher was for Kwint, not you! I will find someone more trustworthy to take it to him.");
		e.other:Ding();
		e.other:Faction(291,-10,0); -- Faction: Merchants of Qeynos
		e.other:Faction(223,5,0); -- Faction: Circle of Unseen Hands
		e.other:Faction(219,-10,0); -- Faction: Antonius Bayle
		e.other:Faction(229,-10,0); -- Faction: Coalition of Tradefolk
		e.other:Faction(262,-10,0); -- Faction: Guards of Qeynos
	end
	item_lib.return_items(e.self, e.other, e.trade)
end
